app/models/user.rb

```diff
@@ -50,7 +50,7 @@ class User < ActiveRecord::Base
   validates :state, inclusion: { in: VALID_STATES,
                                  message: "must be one of #{VALID_STATES.join(',')}" }
 
-  validate :name_part_required_for_suffix_or_title
+  validates :first_name, :last_name, presence: true, allow_blank: false
```

Comment:
Yes that makes sense.  I had to get a bit more complex on the logic to require names on new records, but think what I came up with is the intended behavior